service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/BehandlingStatistikkMapper.kt

```diff
@@ -250,6 +338,17 @@ class BehandlingStatistikkMapper(
                 is UnderkjentRevurdering.IngenEndring -> "Revurdering uten endring i ytelse underkjent"
                 else -> throw ManglendeStatistikkMappingException(this, revurdering::class.java)
             }
+
+        fun map(gjenopptak: GjenopptaYtelseRevurdering): String =
+            when (gjenopptak) {
+                is GjenopptaYtelseRevurdering.IverksattGjenopptakAvYtelse -> "Ytelsen er gjenopptat"
+                is GjenopptaYtelseRevurdering.SimulertGjenopptakAvYtelse -> "Simulert gjenopptak av ytelse"
```

Comment:
```suggestion
                is GjenopptaYtelseRevurdering.SimulertGjenopptakAvYtelse -> "Simulert gjenopptak av ytelsen"
```

Reply:
Bryr statistikk seg om disse simuleringene av stans/gjenoppta? Ramzi merget inn avslutting av revurdering i stad. Det finnes tilsvarende for søknadbehandling med søknad og kun søknad også. Sender vi den statistikken vi skal for disse hendelsene og? :smile:

Reply:
Nei, stønadsstatistikk bryr seg kun om vedtak/endringer i ytelse. De som holder på med saksbehandlingsstatistikk bryr seg kun om "viktige" hendelser i saksbehandlingsflyten, så da kan vi nok holde simulering utenfor.

Reply:
Aha, tok den med fordi dette representerer en sorts "opprettelse" av en revurdering for stans. Per nå så simuleres den direkt ved opprettelse. Men kan fjerne dette eventet hvis det er irrelevant! 😅

Reply:
"Opprettelse av revurdering" er nok en viktig hendelse, som saksbehandlingsstatistikk ønsker ja. Mens "simulering" ikke er det. Men hvis da typen faktisk representerer en "opprettelse av revurdering", så kan det jo være at den skal trigge en sending av noe til statistikk.